tools/mz_to_sqlite/mz_to_sqlite.xml

```diff
@@ -1,4 +1,4 @@
-<tool id="mz_to_sqlite" name="mz to sqlite" version="2.0.4">
+<tool id="mz_to_sqlite" name="mz to sqlite" version="2.1.0">
```

Comment:
can we stick to 2.0.4 but add a suffix `+galaxy1`

Reply:
Sure, for my information what is the meaning of the naming scheme?

Reply:
We have some recommendations here: https://galaxy-iuc-standards.readthedocs.io/en/latest/best_practices/tool_xml.html#tool-versions

The main version should reflect the underlying tool version, to make it less confusing and improve the communication with the admin.